### DISCUSSION TITLE
トラックボールが動作しない

### DISCUSSION THREAD

**Discussion by mannnami** [Yowkees/keyball#141] [Q&A] [answered]
keyball61を購入、８月下旬より使用しています。
昨日までは順調に使用できていたのですが、今日、職場で使用しようとしたところトラックボールが動作しなくなりました。
原因として考えられる事や解決策を教えていただけますでしょうか。
現在の状況・使用環境等は以下の通りです。
・問題があるのはトラックボールのみ
・キーボードやマウスクリックの動作等は正常です
・普段は職場と家で使用しています。職場に行くときはカメラ用バッグに入れてリュックで持ち運んでいます（自動車通勤のため振動・衝撃はそれほどないと思います。）
以上、よろしくお願いいたします。

**Comment by Yowkees** [comment, marked as answer]
トラックボールのみ不調ということで、トラックボールのジャンパ４か所の半田付けをやり直してみてください。

半田付け部分は手はんだに限らず実装機による自動実装であっても、振動や昼夜の温度変化による膨張収縮によるヒートショックにてクラック（ひび）が、遅かれ早かれ起こり導通不良となります。
特にクラックが起きやすいと考えられるのが半田によるジャンパー部分です。
クラックは再度熱を加えれば解消しますので、お試しいただければ幸いです。

**Reply by mannnami** [reply]
トラックボールのジャンパのハンダ付けやり直したところ動くようになりました
ありがとうございました。
